Всем, привет!
А я не могу разрешить этот конфликт, в ветке девелопмент в app.module запущен импорт логера import { LoggerMiddleware } from './common/middleware';
Что выдает мне ошибку - Cannot find module './common/middleware' or its corresponding type declarations.
При попытке коммита даже с этой ошибкой, не пропускает встроенный в коммит линт.

/Users/sveta/Documents/ya-pomogayu/backend/src/app.module.ts
  28:34  error  Missing file extension for "./common/middleware"        import/extensions
  28:34  error  Unable to resolve path to module './common/middleware'  import/no-unresolved

Когда же я дописываю путь import { LoggerMiddleware } from './common/middleware/loggerMiddleware' Он снова выдает конфликт.
Как это поправить?

--- a/src/app.module.ts
+++ b/src/app.module.ts
@@ -25,6 +25,7 @@ import { TasksModule } from './core/tasks/tasks.module';
 import { PolicyModule } from './core/policy/policy.module';
 import { SystemApiModule } from './api/system-api/system-api.module';
 import { LoggerMiddleware } from './common/middleware';
+import { WebsocketApiModule } from './api/websocket-api/websocket-api.module';
 
 @Module({
   imports: [

